### ISSUE TITLE
Add a setting to disable controller input

### ISSUE THREAD

**Issue by TheFeelTrain** [jellyfin/jellyfin-media-player#129]
This is probably an extremely niche problem but I use an Xbox controller to control my mouse cursor among other things with some custom bindings. During video playback (not during music as far as I can tell) the left thumbstick controls seeking so the player will jump around when I try to move my mouse.

I've seen some similar issues over on the jellyfin-web repo but they only fixed input when the window was out of focus, they never actually added an option to outright disable controllers as far as I can tell.

**Comment by iwalton3** [comment]
There is a file like this you can edit: https://github.com/jellyfin/jellyfin-media-player/blob/master/resources/inputmaps/xbox-controller-windows.json

**Comment by TheFeelTrain** [comment]
Hmm that seems to be a carry-over from PMP so I tried looking in that direction.

I found this: https://reddit.com/r/PleX/comments/64ck8s/is_there_a_way_to_disable_xbox_controls_on_plex/dg1gclq/

I tried moving all of the xbox and even dualshock json files out of the examples folder then removing all the mappings. Does not seem to work. I'll investigate further but I'm not sure if this is what is actually getting used.

With some digging it seems to actually be the JS input handling of jellyfin-web. I will close this issue and reopen it over there instead.
